core/src/main/resources/jenkins/appearance/AppearanceGlobalConfiguration/index.jelly

```diff
@@ -48,10 +48,6 @@ THE SOFTWARE.
           </l:notice>
         </j:when>
         <j:otherwise>
-          <div class="behavior-loading">
```

Comment:
any reason you removed from this page? if you throttle to a slower connection its noticeable

Reply:
@janfaracik ?